app/assets/stylesheets/docs.scss

```diff
@@ -0,0 +1,108 @@
+$blue: #53ADC6;
+$red: #CF726E;
+$green: #5BC89E;
+$white: #f0f0f0;
+
+html, body {
+  margin: 0;
+
+  line-height: 1.62em;
+  font-family: 'Lato', serif;
+  font-size: 18px;
+}
+
+img {
+  max-width: 100%;
+}
+
+h1, h2, h3 {
+  margin-top: 2em;
+  font-family: 'Fjalla One', sans-serif;
+}
+
+code {
+  font-size: .8rem;
+  padding: .4em .2em 0;
+  font-family: 'Source Code Pro', ;
+}
+
+a {
+  text-decoration: none;
+  color: $blue;
+}
+
+.content {
+  width: 61em;
```

Comment:
Properties should be ordered margin-bottom, margin-left, margin-right, margin-top, width